guides/common/modules/proc_upgrading-the-external-database.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Upgrading_the_External_Database_{context}"]
+= Upgrading the External Database
+
+You can upgrade an external database from {EL} 7 to {EL} 8 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+
+.Prerequisites
+* Create a new {EL} 8 based host for PostgreSQL server that follows the external database on {EL} 8 documentation.
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/deploying_different_types_of_servers/using-databases#migrating-to-a-rhel-8-version-of-postgresql_using-postgresql[Migrating to a RHEL 8 version of PostgreSQL].
+
+.Procedure
+. On a virtual machine, take a snapshot.
+. On a physical machine, create a backup.
```

Comment:
This should be reduced to:
```suggestion
. Create a backup.
```

This is because the procedure later states "Copy the backup database".

Additionally, many sysadmins will keep a virtual server around for a while as a backup, instead of separate backups/snapshots. In most virtualization platforms you can just shut it down anyway.